modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/exportimport/data/handler/StagedGroupedWorkflowDefinitionLinkStagedModelDataHandler.java

```diff
@@ -98,21 +95,6 @@ protected void doExportStagedModel(
 			String.valueOf(
 				stagedGroupedWorkflowDefinitionLink.
 					getWorkflowDefinitionVersion()));
-
-		if (typePK == -1) {
-			return;
-		}
-
-		DDMStructure ddmStructure = _ddmStructureLocalService.fetchStructure(
-			typePK);
-
-		if (ddmStructure == null) {
-			return;
-		}
-
-		StagedModelDataHandlerUtil.exportReferenceStagedModel(
```

Comment:
Are you sure it is not necessary to export this reference?

If selected structure is not included in LAR file  as a reference (for example in case a structure is from global site), I think it will fail during import operation in external Liferay installation

Reply:
Could you please provide the steps for this? AFAIK the entity that is using the structure is responsible for exporting it, not the WorkflowDefinitionLink

Reply:
Same steps than https://issues.liferay.com/browse/LPS-99881 but instead of creating the structures in same site, you create them in the "Global" site.

I have already left the office today and I am writing from the mobile phone. If you need more information I can have a look to it on Monday

Reply:
I created 2 structures on the global site, then I created a new site, added a folder and configured the workflow for the two structures in the folder, then I exported the folder, created another site and imported successfully, so I think it's good. I'll forward this to BChan, but if you find a scenario in which this change would be necessary, please let me know. Thanks!

Reply:
Hi @inacionery 

I have done some testing and you are right, JournalFolder-DDMStructure dependency is exported by JournalFolder itself, so it is not necessary to add it again in WorkflowDefinitionLink.
By the way, I saw you closed https://github.com/brianchandotcom/liferay-portal/pull/77097 PR, did you missed something?

Thank you

Reply:
There was a weird CI failure, but I've already resend https://github.com/brianchandotcom/liferay-portal/pull/77135